ironfish-cli/src/utils/chainport/requests.ts

```diff
@@ -70,7 +70,13 @@ const makeChainportRequest = async <T extends object>(url: string): Promise<T> =
       return response.data
     })
     .catch((error) => {
-      throw new Error('Chainport error: ' + error)
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      const chainportError = error.response?.data?.error?.description as string
+      if (chainportError) {
```

Comment:
Do we need to update the `ChainportError` type here?

nit - I think the type here is `string | undefined`?